README.md

```diff
@@ -346,6 +337,36 @@ If the column that best matches your system is red, then maybe things aren't
 just broken for you. If it's green, then you can click the "stdio" links in the
 latest build to see what commands the build bots run, and what the output was.
 
+## Building Halide with make
+
+**TL;DR**: Have LLVM 17 (or greater) installed and run `make` in the root
```

Comment:
Maybe "We do not provide support for the Makefile. Feel free to use it, but if anything goes wrong, switch to the cmake build. Note also that the Makefile build will not build the python bindings."

Reply:
Done, with a big, bright [warning box](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts)

Reply:
I left this comment on the individual commit. Repeating here for visibility:

I strongly preferred my suggested wording. "strongly recommend" is vague. If the makefile build works for you, and you use it, nothing's going to explode. The built artifacts are just fine. It's also not "broken at times" - it's tested in CI and you can't merge a PR that breaks it. However it's only tested on specific platforms in specific scenarios, and may simply not work on different machines (usually because it's worse at finding dependencies). My wording also made it clearer that if it doesn't work for you, we will provide zero support, not limited support. We'll just tell you to use the cmake build.

Reply:
Sorry, Andrew! You're right, I didn't see your suggested wording. I updated the warning to a slight revision that emphasizes that the Makefile _cannot_ build the Python bindings or install packages, rather than that it "won't".